examples/dynamo_integration/scripts/plot_throughput_vs_tsq_per_request.py

```
@@ -0,0 +1,874 @@
+        
+        # Create simplified param names (e.g., 'params_llms.dynamo_llm.temperature' -> 'temperature')
+        rename_map = {}
+        for col in param_cols:
+            # Extract the last part after the last '.'
+            simple_name = col.split('.')[-1]
+            rename_map[col] = simple_name
+        
+        df = df.rename(columns=rename_map)
+        
+        # Keep trial_number, value, and simplified param columns
+        keep_cols = ['trial_number', 'value'] + list(rename_map.values())
```

Ruff (0.14.8)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)